test/manual/algo/Dockerfile

```diff
@@ -12,4 +12,4 @@ COPY ./lin_reg.py /cocos/lin_reg.py
 RUN pip install -r requirements.txt
 
 # command to be run when the docker container is started
-CMD ["python3", "/cocos/lin_reg.py"]
+CMD ["python", "-u", "/cocos/lin_reg.py"]
```

Comment:
Can this be done using a different Docker command in Agent, rather than relying on Python?